internal/kafka/internal/handlers/service_accounts.go

```diff
@@ -173,14 +173,14 @@ func (s serviceAccountsHandler) GetServiceAccountById(w http.ResponseWriter, r *
 	handlers.HandleGet(w, r, cfg)
 }
 
-func (s serviceAccountsHandler) GetSsoProvider(w http.ResponseWriter, r *http.Request){
+func (s serviceAccountsHandler) GetSsoProvider(w http.ResponseWriter, r *http.Request) {
```

Comment:
should we also make `GetSsoProvider` to `GetSsoProviders`?

Reply:
This code i.e the method name is not generated that's why it wasn't changed. Let me know if that's the change you'd want to make also and I can modify it.

Reply:
we can keep that way or wdyt?

Reply:
Thanks for raising this. So I checked how are other handlers named and they seem to align with the `operationId` defined in the OpenAPI file. Let's change it and add the `s` too.